libraries/core_libs/network/src/tarcap/taraxa_capability.cpp

```diff
@@ -321,13 +321,29 @@ void TaraxaCapability::interpretCapabilityPacket(std::weak_ptr<dev::p2p::Session
       return;
     }
   }
-
+  SubprotocolPacketType packet_type = static_cast<SubprotocolPacketType>(_id);
+  if (syncing_state_->is_deep_pbft_syncing() && filterSyncIrrelevantPackets(packet_type)) {
+    LOG(log_dg_) << "Ignored " << convertPacketTypeToString(packet_type) << " because we are still syncing";
+    return;
+  }
   // TODO: we are making a copy here for each packet bytes(toBytes()), which is pretty significant. Check why RLP does
   //       not support move semantics so we can take advantage of it...
-  SubprotocolPacketType packet_type = static_cast<SubprotocolPacketType>(_id);
   thread_pool_->push(PacketData(packet_type, std::move(node_id), _r.data().toBytes()));
 }
 
+inline bool TaraxaCapability::filterSyncIrrelevantPackets(SubprotocolPacketType packet_type) const {
+  switch (packet_type) {
+    case TestPacket:
+    case StatusPacket:
+    case GetPbftSyncPacket:
+    case PbftSyncPacket:
+    case SyncedPacket:
```

Comment:
I don't understand why include "TestPacket" and "SatusPacket". During syncing I think still need to update peers status
"GetPbftSyncPacket" is receiving request from peers. Does that mean if node is sending blocks to a peer, the node will not accept other nodes syncing request?
"PbftSyncPacket" is receiving PBFT blocks from peer. node is syncing from peer, but ignore PBFT blocks. Looks like wrong here.
If I am wrong please correct me here. I think the "SyncedPacket" is sender send to receiver the last packet. The receiver node is syncing, but ignore the last received packet. Looks like wrong.
Maybe I am wrong and I misunderstanding here

Reply:
those are packets that we are accepting